doc/math.lyx

```diff
@@ -5084,6 +5084,185 @@ reference "ex:projection"
 
 \end_layout
 
+\begin_layout Subsection
+Derivative of Adjoint
```

Comment:
Did you read section 3 in detail? In this document we are trying to take a different view, eschewing the partial derivative notation. I’m happy to spend some time trying to rewrite it with you…

Reply:
I will re-read in more detail.

I think the "eschewing the partial derivative notation" direction was lost on me based on the very first paragraph of "Part II: Practice" which states:
![image](https://user-images.githubusercontent.com/25356161/137597499-69586177-2581-4346-8409-c4a96d00e14b.png)

which made me assume that this was the preferred notation.  Perhaps we might add a note that this is _not_ the preferred way of thinking about things?

I will go through section 3 and try to emulate that style more, and send you an email when revised to find some time to go over this making sure I have the right ideas etc.